src/routes/report.tsx

```diff
@@ -11,11 +20,30 @@ export interface ReportRouteProps {
   env?: string;
 }
 
-function ReportRoute(props: ReportRouteProps): VNode {
+function ReportRoute({
+  path,
+  start: startParam,
+  finish: finishParam,
+  ial: ialParam,
+  agency,
+  env: envParam,
+}: ReportRouteProps): VNode {
+  const endOfPreviousWeek = utcWeek.floor(new Date());
+  const startOfPreviousWeek = utcWeek.floor(new Date(endOfPreviousWeek.valueOf() - 1));
+
+  const start = (startParam && yearMonthDayParse(startParam)) || startOfPreviousWeek;
+  const finish = (finishParam && yearMonthDayParse(finishParam)) || endOfPreviousWeek;
+  const ial = (parseInt(ialParam || "", 10) || DEFAULT_IAL) as 1 | 2;
+  const env = envParam || DEFAULT_ENV;
+  const [allAgencies, setAllAgencies] = useState([] as string[]);
+
   return (
-    <ReportFilterControls {...props}>
+    <ReportFilterControlsContext.Provider
+      value={{ path, start, finish, ial, agency, env, setAllAgencies }}
```

Comment:
I kinda hoped we wouldn't have to pass around `path` but instead pull it directly from `preact-router`'s internal context. I saw there's an [exported `useRouter`](https://github.com/preactjs/preact-router/blob/24ffb40c8df2a455bc9518aa0ffdb7c5a1878968/src/index.js#L270) which would give us access to this, but it's not yet published 😞

Reply:
😩

Reply:
Well I decided that `path` didn't need to live in the filter context so I did move it out in fe3169f9b30a1f3d0b3ba8ad9073ca7406b48f35

Reply:
> Well I decided that `path` didn't need to live in the filter context so I did move it out in [fe3169f](https://github.com/18F/identity-reporting/commit/fe3169f9b30a1f3d0b3ba8ad9073ca7406b48f35)

That does seem better 👍